openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -198,25 +198,36 @@
   narrative:
     - key: a
       text: |
-        Organizational review and analysis of information system
-        audit records for indications of organization-defined
-        inappropriate or unusual activity is outside the scope
-        of OpenShift configuration.
+        While OpenShift can generate audit data, and securely transport
+        the data to a SIEM, OpenShift does not provide automated mechanisms
+        (e.g. email, text messages) after system events. This functionality
+        would fall to third party software such as Splunk, Elastic, etc.
+        
+        To configure an IdP, see:
+        https://docs.openshift.com/container-platform/latest/authentication/identity_providers/configuring-ldap-identity-provider.html#configuring-ldap-identity-provider
     - key: b
       text: |
-        Reporting findings of organization-defined inappropriate or
-        unusual activity to organization-defined personnel or roles is
-        outside the scope of OpenShift configuration.
+        While OpenShift can generate audit data, and securely transport
+        the data to a SIEM, OpenShift does not provide automated mechanisms
+        (e.g. email, text messages) after system events. This functionality
+        would fall to third party software such as Splunk, Elastic, etc.
+        
+        To configure an IdP, see:
+        https://docs.openshift.com/container-platform/latest/authentication/identity_providers/configuring-ldap-identity-provider.html#configuring-ldap-identity-provider
 
 - control_key: AU-6 (1)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-167
+        While OpenShift can generate audit data, and securely transport
+        the data to a SIEM, OpenShift does not provide automated mechanisms
+        (e.g. email, text messages) after system events. This functionality
+        would fall to third party software such as Splunk, Elastic, etc.
+        
+        To configure an IdP, see:
+        https://docs.openshift.com/container-platform/latest/authentication/identity_providers/configuring-ldap-identity-provider.html#configuring-ldap-identity-provider
```

Comment:
Language of AU-6(1) can be found at https://nvd.nist.gov/800-53/Rev4/control/Au-6#enhancement-1:


AU-6(1) | AUDIT REVIEW, ANALYSIS, AND REPORTING \| PROCESS INTEGRATIONThe organization employs automated mechanisms to integrate audit review, analysis, and reporting processes to support organizational processes for investigation and response to suspicious activities.
-- | --

When triaging with Engineering, the general thought was to defer this control to a 3rd party SIEM. Currently, OpenShift logging operators/dashboards (eg Kibana, Elastic, etc) cannot be configured to provide this capability. There was a product Jira ticket opened so that customers could attach RFEs & allow us to measure if customers are OK with deferring to 3rd party or want OCP to have this functionality natively.

Reply:
If OpenShift is updated at some point to include functionality to support this, then we address.  For now, it is deferred to 3rd party.